### PR TITLE
cancelAppointments

### LINKED ISSUE #76: Add button to cancel appointment
At the end, when we show the appointment details, there should be a button that allows them to cancel the appointment.

### PR DESCRIPTION
**Pull request by jerometujague**
Made a new branch with the requested changes, I deleted the old one because I wasn't sure what was causing the errors. 
However, after adding the new code, when I click the initial schedule appointment button, it just stays on that screen and doesn't show the details page. Checking the console shows an unhandled promise rejection error. Any idea what is causing this? I'm sure it is something simple that I missed. 
Closes #76

### CODE REVIEW

File: src/main/java/com/commercebank/controller/AppointmentController.java [hunk 1]
```diff
@@ -60,6 +62,11 @@ public AppointmentController(AppointmentDAO appointmentDAO, CustomerDAO customer
         return appointmentDAO.list();
     }
 
+    @RequestMapping(value = "/delete/{id}/", method = RequestMethod.GET)
```
Comment (jmbockhorst) [inline src/main/java/com/commercebank/controller/AppointmentController.java]: Is there a reason you changed this from a DELETE request?

Comment (jerometujague) [reply src/main/java/com/commercebank/controller/AppointmentController.java]: No, delete didn't work either so I was checking to see if GET worked. I just changed it back to DELETE.

**Comment by jmbockhorst** [comment]
Can you screenshot the console error? I’m not at home so I can’t test it.

**Comment by jmbockhorst** [comment]
Also, check the IntelliJ log for any errors.

**Comment by jerometujague** [comment]
Screen Shot 2019-03-27 at 8 18 43 PM
Screen Shot 2019-03-27 at 8 19 00 PM

These are the console error and the error log in IntelliJ.

**Comment by jmbockhorst** [comment]
It’s something to do with AppointmentController line 148, the index is out of bounds. Re initialize your database.

**Comment by jmbockhorst** [comment]
It's a bug that we need to fix, but for now, re running the database script should fix it.

**Comment by jerometujague** [comment]
Okay. We can work on this more later.

**Comment by jmbockhorst** [comment]
I got it to work for me. After it deletes it successfully, we need to show another page saying that it was deleted.

**Comment by jerometujague** [comment]
Works for me as well if I make sure the customer ids are in numerical order. I will add the deleted confirmation page. 
Do you think we should send another email confirming that the appointment was deleted?

**Comment by jmbockhorst** [comment]
> Do you think we should send another email confirming that the appointment was deleted?

I don't think its a bad idea. We can ask the rest of the team what they think.

**Comment by jmbockhorst** [comment]
Something else I think we should add is a confirmation dialog when they try to cancel, but that can be added later.

**Pull request merged**